development/openstack-converged-networking-focal-ussuri/README.md

```diff
@@ -0,0 +1,43 @@
+# OpenStack Converged Networking
+
+*DEV/TEST ONLY*: This unstable, development example bundle extends the basic OpenStack Cloud bundle with Telemetry collection via Ceilometer. See also: [Stable Bundles](https://jujucharms.com/u/openstack-charmers).
+
+Certain configuration options within the bundle may need to be adjusted prior to deployment to fit your particular set of hardware. For example, network device names and block device names can vary, and passwords should be yours.
+
+For full details on the base cloud deployment please refer to the [Basic OpenStack Cloud][] bundle.
```

Comment:
I think this section should explain the purpose of the OpenStack Converged Networking a little bit here. i.e. This bundle is used to test and demonstrate a baremetal deployment of OpenStack on MAAS using a single network card for the host. This is done by configuring the networking of node in MAAS to use an Openvswitch ...